server/services/echo_service_test.go

```diff
@@ -39,7 +39,10 @@ func TestEcho_success(t *testing.T) {
 
 	server := NewEchoServer()
 	for _, val := range table {
-		in := &pb.EchoRequest{Response: &pb.EchoRequest_Content{Content: val}}
+		in := &pb.EchoRequest{
+			Response: &pb.EchoRequest_Content{Content: val},
+			Severity: pb.Echo_CRITICAL,
```

Comment:
This is incorrect, the regenerated code creates an enum named `Severity_CRITICAL`. Please follow the instructions for setting up the [generation environment](https://github.com/googleapis/gapic-showcase#development-environment). Then, please regenerate the protobuf/gRPC package via `go run ./util/cmd/compile_protos`.